yarn.lock

```diff
@@ -6647,7 +6647,7 @@ babel-plugin-emotion@^8.0.10:
     source-map "^0.5.7"
     touch "^1.0.0"
 
-babel-plugin-emotion@^9.0.0, babel-plugin-emotion@^9.0.1, babel-plugin-emotion@^9.2.11:
+babel-plugin-emotion@^9.0.0, babel-plugin-emotion@^9.2.11:
```

Comment:
We updated this plugin to `10.0.33`... why does the lock file still show this?

Reply:
It should be this right?
<img width="965" alt="Screenshot 2020-05-26 at 7 15 35 PM" src="https://user-images.githubusercontent.com/8639235/82908214-4aa4c300-9f85-11ea-90f4-506a803a13e2.png">